lib/diagram.dx

```diff
@@ -141,32 +141,33 @@ def optionalHtmlColorStr (c: Maybe HtmlColor) : String =
 
 @noinline
 def attrString (attr:GeomStyle) : String =
-  (  --  makeAttr (str "stroke")       (optionalHtmlColorStr $ getAt #strokeColor attr)
-      makeAttr (str "fill")         (optionalHtmlColorStr $ getAt #fillColor   attr)
-  <+> makeAttr (str "stroke-width") (show                 $ getAt #strokeWidth attr))
+  (  -- (str "stroke") <=> (optionalHtmlColorStr $ getAt #strokeColor attr)
+      (str "fill") <=> (optionalHtmlColorStr $ getAt #fillColor   attr)
```

Comment:
At this point I'm wondering if we should default all string literals to `String` type (i.e. the length erased list), and then simply have multiple modifiers (like Python raw or f-strings) that lets you interpret them as e.g. arrays of characters with lengths exposed.

Reply:
I wondered about that too. Does that change carry any meaningful performance implications? Maybe the implications don't matter yet because string performance doesn't matter so much yet in Dex? (or maybe it does matter, for plotting performance)

Reply:
That change has been made in  #367 
https://github.com/google-research/dex-lang/pull/367/files#diff-7352b0160734bf8ae9c7f0c87b1ff7641db108b16e448fa3393a1804ce6c1b3fR222